feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -66,13 +67,13 @@ fun PreviewScreen(
 
     val lifecycleOwner = LocalLifecycleOwner.current
 
-    val deferredSurfaceProvider = CompletableDeferred<SurfaceProvider>()
+    val deferredSurfaceProvider by remember { mutableStateOf(CompletableDeferred<SurfaceProvider>())}
```

Comment:
nit: this probably isn't a mutable state right?
```
val deferredSurfaceProvider = remember { CompletableDeferred<SurfaceProvider>() }
```

Reply:
Ah yes, fixed.